File: BondClient/Models/WorkspaceFile.cs
using System.Text.Json.Serialization;

namespace BondClient.Models;

public class WorkspaceFile
{
    [JsonPropertyName("id")]
    public long Id { get; set; }

    [JsonPropertyName("workspaceId")]
    public long WorkspaceId { get; set; }

    [JsonPropertyName("fileName")]
    public string FileName { get; set; } = "";

    [JsonPropertyName("fileSize")]
    public long FileSize { get; set; }

    [JsonPropertyName("minioPath")]
    public string MinioPath { get; set; } = "";

    [JsonPropertyName("contentType")]
    public string? ContentType { get; set; }

    [JsonPropertyName("uploadedBy")]
    public long UploadedBy { get; set; }

    [JsonPropertyName("parentPath")]
    public string ParentPath { get; set; } = "/";

    [JsonPropertyName("isDirectory")]
    public bool IsDirectory { get; set; }

    [JsonPropertyName("createdAt")]
    public long? CreatedAt { get; set; }
}

File: BondServer/bond-common/src/main/java/com/bond/common/entity/WorkspaceFile.java
package com.bond.common.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@TableName("workspace_files")
public class WorkspaceFile {
    @TableId(type = IdType.ASSIGN_ID)
    private Long id;
    private Long workspaceId;
    private String fileName;
    private Long fileSize;
    private String minioPath;
    private String contentType;
    private Long uploadedBy;
    private String parentPath;
    private Boolean isDirectory;
    private LocalDateTime createdAt;
}
